src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -436,18 +462,26 @@ def _query_workspace_with_load_id_mass(
     load_id: str,
     obj_names: list[str],
     batch_size: int = _WS_MAX_BATCH_SIZE,
-) -> tuple[list[str], list[str]]:
-    uploaded_obj_names = []
-    uploaded_obj_upas = []
-
-    for idx in range(0, len(obj_names), batch_size):
-        obj_names_batch, obj_upas_batch = _query_workspace_with_load_id(
-            ws, workspace_id, load_id, obj_names[idx: idx + batch_size]
-        )
-        uploaded_obj_names.extend(obj_names_batch)
-        uploaded_obj_upas.extend(obj_upas_batch)
-
-    return uploaded_obj_names, uploaded_obj_upas
+    assembly_objs_only: bool = True,
+) -> tuple[list[str], list[str], list[str], list[str]]:
+
+    # TODO: run _query_workspace_with_load_id in parallel instead of in serial
```

Comment:
Just checking - it's serial right now, right? We're not downgrading here?

Reply:
oh. I mean the `for` loop. Parallel within `_query_workspace_with_load_id`.

Reply:
Sorry, not grokking the response here - is there a part missing?

Reply:
hmm I got confused with your question now too.

Reply:
There's a TODO to parallelize something. I'm wondering if that's because it just needs to be parallelized or because this change broke the current parallelization

Reply:
oh I see. Parallelization refers to `for i in range(0, len(obj_names), batch_size)` to bump performance. We don't need to do anything if performance is good.

Reply:
Ok, so this is just a note about a potential future improvement

Reply:
yea. removed the todo. Probably don't need to worry about performance for now.

Reply:
:+1: